src/Compilers/CSharp/Test/Emit/PDB/CheckSumTest.cs

```diff
@@ -309,23 +307,18 @@ class C { void M() { } }
             comp.VerifyPdb(@"
 <symbols>
   <files>
-    <file id=""1"" name=""a\..\a.cs"" language=""C#"" checksumAlgorithm=""406ea660-64cf-4c82-b6f0-42d48172a799"" checksum=""AB-00-7F-1D-23-D5"" />
-    <file id=""2"" name=""C:\a\..\b.cs"" language=""C#"" checksumAlgorithm=""SHA1"" checksum=""36-39-3C-83-56-97-F2-F0-60-95-A4-A0-32-C6-32-C7-B2-4B-16-92"" />
+    <file id=""1"" name=""C:\a\..\b.cs"" language=""C#"" checksumAlgorithm=""SHA1"" checksum=""36-39-3C-83-56-97-F2-F0-60-95-A4-A0-32-C6-32-C7-B2-4B-16-92"" />
+    <file id=""2"" name=""a\..\a.cs"" language=""C#"" checksumAlgorithm=""406ea660-64cf-4c82-b6f0-42d48172a799"" checksum=""AB-00-7F-1D-23-D5"" />
```

Comment:
Why is there no checksum algorithm here?

Reply:
It seems like the checksum algorithm is present on this line. Am I misunderstanding something?

Reply:
ugg, the double quoting fooled me.